cartridges/int_stickyio_sfra/cartridge/scripts/subscription/subscriptionHelpers.js

```diff
@@ -153,6 +153,7 @@ function getSubscriptions(currentCustomer, querystring, locale, billingModels) {
         var thisOrder = OrderMgr.getOrder(orders[i].orderNumber, orders[i].orderToken);
         customerID = thisOrder.getCustomer().ID; // this gets overwritten every time, but that's ok
         var orderShipments = thisOrder.getShipments();
+        orderModel = orders[i];    
```

Comment:
remove the extra spaces at the end of the line please

Reply:
done